File: ProAgil.WebAPI/Model/Evento.cs
using System;

namespace ProAgil.WebAPI.Model
{
    public class Evento
    {
        public int EventoId { get; set; }
        public String Local { get; set; }
        public String DataEvento { get; set; }
        public String Tema { get; set; }
        public int QntPessoas { get; set; }
        public String Lote { get; set; }
        public String ImagemURL { get; set; }

    }
}

File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { Palestrante } from './Palestrante';
import { RedeSocial } from './RedeSocial';

export interface Evento {
    eventoId: number;
    local: string;
    dataEvento: Date;
    tema: string;
    qntPessoas: number;
    imagemURL: string;
    telefone: string;
    email: string;
    lote: string;
    // lotes: Lote[];
    // redesSociais: RedeSocial[];
    // palestrantesEventos: Palestrante[];
}
